runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -152,11 +165,24 @@
 
 	void reset()
 	{
+		setCanvasState(false);
 		container.remove(statsPanel);
 		panel.remove(this);
 		panel.revalidate();
 	}
 
+	void setCanvasState(boolean added)
```

Comment:
this can be private. Also, it probably do not even needs to be separate method.

Reply:
Yeah, had plans to use this method in a different way then I discarded the plans and forgot about the uselesness of the method.